repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/examples/container.cfg.xml

```diff
@@ -32,4 +32,10 @@
 -->
         </ssl-configuration>
     </deployment-config>
+
+    <cluster-config cluster-id="internal-cluster">
```

Comment:
Yeah, there's nothing right now that's really paying attention to both, and even if there was config ordering might be wrong. I'm just not a fan of silent failure, but there's not a great solution to that problem right now. Config read ordering is something we should consider for 9.